docs/UserGuide.md

```diff
@@ -259,9 +262,9 @@ The command `list_session` will show the following:
 * `find_session Zach` returns nothing-->
 <!-- END OF COMMENT OUT FOR FIND SESSION -->
 
-### Adding a tuition session: `add_session`
+### Adding a single tuition session: `add_session`
 
-Adds a tuition session to the TutorBuddy
+Adds a single tuition session to the TutorBuddy
```

Comment:
Hi, just wondering, is "to the TutorBuddy" quite werid, I tink its copied over from single session also. I think changing over to "to TutorBuddy" removing "the" would be better. What do you think?

Reply:
Sounds good. I'll change it on my end.